merino/jobs/navigational_suggestions/__init__.py

```diff
@@ -0,0 +1,101 @@
+"""CLI commands for the navigational_suggestions module"""
+import json
+import logging
+
+import typer
+
+from merino.config import settings as config
+from merino.jobs.navigational_suggestions.domain_data_downloader import (
+    DomainDataDownloader,
+)
+from merino.jobs.navigational_suggestions.domain_metadata_extractor import (
+    DomainMetadataExtractor,
+)
+from merino.jobs.navigational_suggestions.domain_metadata_uploader import (
+    DomainMetadataUploader,
+)
+
+logger = logging.getLogger(__name__)
+
+job_settings = config.jobs.navigational_suggestions
+
+# Options
+source_gcp_project_option = typer.Option(
+    job_settings.source_gcp_project,
+    "--src-gcp-project",
+    help="GCP project to use for downloading the domain data",
+)
+
+destination_gcp_project_option = typer.Option(
+    job_settings.destination_gcp_project,
+    "--dst-gcp-project",
+    help="GCP project to use to store the domain metadata on GCS for navigational suggestions",
+)
+
+destination_gcs_bucket_option = typer.Option(
+    job_settings.destination_gcs_bucket,
+    "--dst-gcs-bucket",
+    help="GCS bucket where the domain metadata for navigational suggestions will be stored",
+)
+
+navigational_suggestions_cmd = typer.Typer(
+    name="navigational-suggestions",
+    help="Command for preparing top domain metadata for navigational suggestions",
+)
+
+
+def _construct_top_picks(
+    domain_data: list[dict],
+    favicons: list[str],
+    urls_and_titles: list[dict],
+    second_level_domains: list[str],
+):
```

Comment:
FYI, we encourage folks to adopt type annotation for Merino so that static type analyzers such as Mypy can help us capture typing errors sooner. 
```suggestion
) -> str:
```

Reply:
Hey @ncloudioj This is great. I had added type annotation everywhere but missed this one 👍🏾